lib/src/features/main/theme/theme_data/app_theme_data.dart

```diff
@@ -1,283 +1,62 @@
-import 'package:flutter/material.dart';
+import 'package:fluent_ui/fluent_ui.dart';
 import 'package:gui/src/core/enums/theme_modes.dart';
 import 'package:gui/src/core/utils/gen/assets/fonts.gen.dart';
 import 'package:gui/src/features/main/theme/theme_data/pallets/on_surface_pallet.dart';
 import 'package:gui/src/features/main/theme/theme_data/pallets/surface_pallet.dart';
 
 /// ## [AppThemeData] Documentation
 ///
-/// The [AppThemeData] class provides theming configurations for the
-/// application.
-/// It contains a static map that holds the `ThemeData` for both light and dark
-/// modes, allowing for easy theme management based on the user's preference.
+/// The `AppThemeData` class provides theming configs for the application,
+/// supporting both light and dark themes.
 ///
+/// ### Properties
+///
+/// - **`themeDataModes`**: A static map containing theme configurations for:
+///   - **Light Theme**: Configured with `lightTypography` and light palette.
+///   - **Dark Theme**: Configured with `darkTypography` and dark palette.
+///
+/// - **`lightTypography`**: The typography config used for the light theme.
+///
+/// - **`darkTypography`**: The typography config used for the dark theme.
+///
+/// ### Example Usage
+///
+/// ```dart
+/// final theme = AppThemeData.themeDataModes[ThemeModes.light];
+/// ```
 class AppThemeData {
   const AppThemeData._();
-  static final Map<ThemeModes, ThemeData> themeDataModes = {
+
+  static final Map<ThemeModes, FluentThemeData> themeDataModes = {
     // light ThemeData.
-    ThemeModes.light: ThemeData(
-      bottomNavigationBarTheme: BottomNavigationBarThemeData(
-        backgroundColor: SurfacePallet.light.surface3,
-      ),
+    ThemeModes.light: FluentThemeData(
+      fontFamily: FontFamily.inter,
       scaffoldBackgroundColor: SurfacePallet.light.surface3,
       brightness: Brightness.light,
-      dividerColor: OnSurfacePallet.light.onSurface1,
-      textTheme: lightTextTheme,
+      typography: lightTypography,
       cardColor: SurfacePallet.light.surface3,
       extensions: const <ThemeExtension<dynamic>>[
         OnSurfacePallet.light,
         SurfacePallet.light,
       ],
     ),
-
     // dark ThemeData.
-    ThemeModes.dark: ThemeData(
+    ThemeModes.dark: FluentThemeData(
+      fontFamily: FontFamily.inter,
+      scaffoldBackgroundColor: SurfacePallet.dark.surface3,
       brightness: Brightness.dark,
-      textTheme: darkTextTheme,
-      extensions: <ThemeExtension<dynamic>>[
+      typography: Typography.raw(),
+      cardColor: SurfacePallet.dark.surface3,
+      extensions: const <ThemeExtension<dynamic>>[
         OnSurfacePallet.dark,
         SurfacePallet.dark,
       ],
     ),
   };
 
```

Comment:
@esmaeil-ahmadipour , one suggestion I have is that the UI designer create these text styles based on the Fluent UI typography standards found in this [link ](https://fluent2.microsoft.design/typography)and seek help from them.

Reply:
this item explained in this [commit](https://github.com/pactus-project/pactus-gui/issues/45#issuecomment-2563754458)  and will be resolve in the issue #45 .